plat/kvm/x86/setup.c

```diff
@@ -273,15 +273,23 @@ static __sz cmdline_len;
 
 static inline int cmdline_init(struct ukplat_bootinfo *bi)
 {
-	char *cmdl = (bi->cmdline) ? (char *)bi->cmdline : CONFIG_UK_NAME;
+	char *cmdl;
 
-	cmdline_len = strlen(cmdl) + 1;
+	if (bi->cmdline) {
```

Comment:
Perhaps it's better to use `bi->cmdline_len` here, as there is a chance that in the case of no args, `bi->cmdline` will point to a `NUL` byte.